OpenRA.Game/Game.cs

```diff
@@ -823,7 +840,8 @@ static void Loop()
 						LogicTick();
 
 						// Force at least one render per tick during regular gameplay
-						if (OrderManager.World != null && !OrderManager.World.IsLoadingGameSave && !OrderManager.World.IsReplay)
+						// TODO: Have OrderManager tell us when we should force render
```

Comment:
!

Reply:
I intended this to mean TODO in a later PR, removing